lib/fix_db_schema_conflicts/tasks/db.rake

```diff
@@ -1,19 +1,13 @@
 require 'shellwords'
 require_relative '../autocorrect_configuration'
 
-namespace :db do
-  namespace :schema do
-    task :dump do
-      puts "Dumping database schema with fix-db-schema-conflicts gem"
+Rake::Task['db:schema:dump'].enhance do
+  puts 'Dumping database schema with fix-db-schema-conflicts gem'
 
-      filename = ENV['SCHEMA'] || if defined? ActiveRecord::Tasks::DatabaseTasks
-        File.join(ActiveRecord::Tasks::DatabaseTasks.db_dir, 'schema.rb')
-      else
-        "#{Rails.root}/db/schema.rb"
-      end
-      autocorrect_config = FixDBSchemaConflicts::AutocorrectConfiguration.load
-      rubocop_yml = File.expand_path("../../../../#{autocorrect_config}", __FILE__)
-      `bundle exec rubocop --auto-correct --config #{rubocop_yml} #{filename.shellescape}`
-    end
-  end
+  # Copied from active_record/railties/databases.rake
+  filename = ENV['SCHEMA'] || File.join(ActiveRecord::Tasks::DatabaseTasks.db_dir, 'schema.rb')
+
+  autocorrect_config = FixDBSchemaConflicts::AutocorrectConfiguration.load
+  rubocop_yml = File.expand_path("../../../../#{autocorrect_config}", __FILE__)
+  `bundle exec rubocop --auto-correct  --format quiet --config #{rubocop_yml} #{filename.shellescape}`
```

Comment:
Added the `--format quiet` option. Makes little difference. I had hoped that it would silence the warnings related the `rubocop-performance` gem... but it didn't.